src/porepy/numerics/ad/operator_functions.py

```diff
@@ -9,97 +16,90 @@
 
 import abc
 from functools import partial
-from typing import Callable, Optional, Type
+from typing import Callable, Optional, Sequence, Type
 
 import numpy as np
 import scipy.sparse as sps
 
 import porepy as pp
 from porepy.numerics.ad.forward_mode import AdArray
 
+from .functions import FloatType
 from .operators import Operator
 
 __all__ = [
-    "Function",
-    "ConstantFunction",
```

Comment:
ConstantFunction was removed, since this is a duplicate functionality of Wrapped array or scalar.